Build.fsx

```diff
@@ -69,7 +69,7 @@ Target "PatchAssemblyVersions" (fun _ ->
         buildVersion.fileVersion
         buildVersion.infoVersion
 
-    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.*"
+    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.fs"
```

Comment:
See [this](https://github.com/AutoFixture/AutoFixture/pull/799/commits/643c9d1daf234f40380867f29f8c1293eadbaea1) commit and its message. In short - we don't need it more as there is nothing to patch in `AssemblyInfo.cs` files 😄